### PR TITLE
Selected translations

### PR DESCRIPTION
**Pull request by teolemon**

### CODE REVIEW

File: po/common/fr.po [hunk 1]
```diff
@@ -1134,7 +1134,7 @@ msgstr "Les produits qui possèdent le label %s"
 
 msgctxt "labels_s"
 msgid "label"
-msgstr "label"
```
Comment (stephanegigandet) [inline po/common/fr.po:1137]: that looks like a bug

Comment (teolemon) [reply po/common/fr.po:1137]: why ?

File: po/common/fr.po [hunk 2]
```diff
@@ -1134,7 +1134,7 @@ msgstr "Les produits qui possèdent le label %s"
...
+msgstr ""
```
Comment (teolemon) [inline po/common/fr.po:1137]:
```suggestion
msgstr "label"
```

File: po/common/fr.po (hunk 1 above)
Comment (teolemon) [reply po/common/fr.po:1137]: I'm fixing it, but that should have no impact

File: po/common/fr.po [hunk 3]
```diff
@@ -2451,7 +2451,7 @@ msgstr "Indiquer les ingrédients des mentions \"Peut contenir des traces de\",
 
 msgctxt "traces_p"
 msgid "traces"
-msgstr "traces"
+msgstr ""
```
Comment (teolemon) [inline po/common/fr.po:2454]:
```suggestion
msgstr "traces"
```

File: po/common/fr.po [hunk 4]
```diff
@@ -3772,7 +3772,7 @@ msgstr "import"
 
 msgctxt "imports"
 msgid "imports"
-msgstr "imports"
+msgstr ""
```
Comment (teolemon) [inline po/common/fr.po:3775]:
```suggestion
msgstr "imports"
```

File: po/common/fr.po [hunk 5]
```diff
@@ -4401,7 +4401,7 @@ msgstr "Les photos du produits peuvent aussi être fournies séparément, via la
 
 msgctxt "attribute_group_labels_name"
 msgid "Labels"
-msgstr "Labels"
+msgstr ""
```
Comment (teolemon) [inline po/common/fr.po:4404]:
```suggestion
msgstr "Labels"
```

**Pull request closed without merging**